CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ To see all merged commits on the master branch that will be part of the next plo
 
 where X.Y.Z is the semver of most recent plotly.js release.
 
+## [next-major]
+
```

Comment:
Instead of modifying the CHANGELOG directly, please add a file named `draftlogs/7017_change.md` as described here: https://github.com/plotly/plotly.js/blob/master/draftlogs/README.md.
Once the PR is merged these files are used for the CHANGELOG.